### ISSUE TITLE
Enhancement: expose deck autoplay media feature in prefs (or remove it)

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#4940]
There is a lot of infrastructure in the code for a preference per deck that lets you autoplay media or not, but it doesn't appear to be actually exposed in the UI anywhere.

Here's the resource handle (that is seemingly correctly plumbed through everywhere including to backing storage, just not in the deck options prefs screen):

AnkiDroid/src/main/res/xml/deck_options.xml:                android:title="@string/deck_conf_autoplay" />

**Comment by timrae** [comment]
```xml
            <CheckBoxPreference
                android:key="autoPlayAudio"
                android:title="@string/deck_conf_autoplay" />
```

```
<string name="deck_conf_autoplay">Automatically play audio</string>
```

**Comment by timrae** [comment]
I'm not sure how you came to the conclusion that it's not exposed... It's right there in Deck options -> General

**Comment by mikehardy** [comment]
I must have been going too fast while looking to fix the 27.1 support issue and attempting to log the console notice about 'data:' URL navigation - I should have looked through all the deck options...
